app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="dev-perl/Crypt-CBC
+	dev-perl/Device-SerialPort
+	dev-perl/Digest-CRC
+	dev-perl/JSON"
+
+DEPEND="media-gfx/pngcrush"
+
+DOCS=( "CHANGED" "HISTORY" "README.SVN" "README_DEMO.txt" "docs"/*.txt "docs"/*.patch "docs"/*.pdf "docs/changelog" "docs/copyright" "docs/dotconfig" "docs/fhem.odg.readme" "docs/LIESMICH.update-thirdparty" "docs"/README* "docs/X10" )
```

Comment:
Globs in global scope are invalid. I wonder why it didn't explode on you.

Reply:
Hm, it worked perfectly fine, but moved to a local scope.